astronomer/providers/amazon/aws/example_dags/example_batch.py

```diff
@@ -0,0 +1,274 @@
+
+    except ClientError as error:
+        logging.exception("Error while disabling Batch Compute Environment")
+        raise error
+
+
+def delete_job_queue_func() -> None:
+    """Delete Batch Job Queue"""
+    import boto3
+    from botocore.exceptions import ClientError
+
+    client = boto3.client("batch")
+    try:
+        client.delete_job_queue(
+            jobQueue=JOB_QUEUE,
+        )
+        while get_job_queue_status() == "DELETING":
+            logging.info("Waiting for job queue to be DELETED. Sleeping for 30 seconds.")
+            time.sleep(30)
+
+    except ClientError as error:
+        logging.exception("Error while deleting Batch Job Queue")
+        raise error
+
+
+with DAG(
+    dag_id="example_async_batch",
+    schedule_interval=None,
+    start_date=datetime(2021, 1, 1),
+    tags=["example", "async", "AWS"],
+    catchup=False,
+) as submit_dag:
+    # Task steps for DAG to be self-sufficient
+    setup_aws_config = BashOperator(
+        task_id="setup_aws_config",
+        bash_command=f"aws configure set aws_access_key_id {AWS_ACCESS_KEY_ID}; "
+        f"aws configure set aws_secret_access_key {AWS_SECRET_ACCESS_KEY}; "
+        f"aws configure set default.region {AWS_DEFAULT_REGION}; ",
+    )
+
+    # Task to create Batch compute environment
+    create_batch_compute_environment = PythonOperator(
+        task_id="create_batch_compute_environment",
+        python_callable=create_batch_compute_environment_func,
+    )
+
+    # Task to create Batch Job Queue
+    create_job_queue = PythonOperator(
+        task_id="create_job_queue",
+        python_callable=create_job_queue_func,
+    )
+
+    # [START howto_operator_batch]
+    submit_batch_job = BatchOperatorAsync(
+        task_id="submit_batch_job",
+        job_name=JOB_NAME,
+        job_queue=JOB_QUEUE,
+        job_definition=JOB_DEFINITION,
+        overrides=JOB_OVERRIDES,
+        aws_conn_id=AWS_CONN_ID,
+        region_name=AWS_DEFAULT_REGION,
+    )
+    # [END howto_operator_batch]
+
+    update_compute_environment = PythonOperator(
+        task_id="update_compute_environment",
+        python_callable=update_compute_environment_func,
+        trigger_rule="all_done",
+    )
+
+    update_job_queue = PythonOperator(
+        task_id="update_job_queue", python_callable=update_job_queue_func, trigger_rule="all_done"
+    )
+
+    delete_job_queue = PythonOperator(
+        task_id="delete_job_queue", python_callable=delete_job_queue_func, trigger_rule="all_done"
+    )
+
+    delete_compute_environment = PythonOperator(
+        task_id="delete_compute_environment",
+        python_callable=delete_compute_environment_func,
+        trigger_rule="all_done",
+    )
+
+    (
+        setup_aws_config
+        >> create_batch_compute_environment
+        >> create_job_queue
+        >> submit_batch_job
+        >> update_compute_environment
+        >> update_job_queue
+        >> delete_job_queue
+        >> delete_compute_environment
```

Comment:
if the task `delete_compute_environment` will succeed airflow will mark DAG as a success so what about creating a dummy task `end` which with below dependency 

```
[update_compute_environment, update_job_queue, delete_job_queue, delete_compute_environment] >> end
```

Reply:
changed in [e10446b](https://github.com/astronomer/astronomer-providers/pull/404/commits/e10446b11b34f47b670ecf8f77f19471ac292221)